### PR TITLE
Editorial: link up implementation defined

### LINKED ISSUE #82: Nit: clearly indicate implementation-defined behavior
By xref'ing the term implementation-defined in Infra.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #82

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/geolocation-api/pull/86.html" title="Last updated on Jun 10, 2021, 4:54 AM UTC (6ca2be2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/geolocation-api/86/f005a86...6ca2be2.html" title="Last updated on Jun 10, 2021, 4:54 AM UTC (6ca2be2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -747,9 +747,9 @@ <h2>
             </ol>
           </li>
           <li>Wait for a significant change of geographic position. What
-          constitutes a significant change of geographic position is left to
-          the implementation. A user agents MAY impose a rate limit on the
-          frequency position changes.
+          constitutes a significant change of geographic position is
+          [=implementation-defined|left to the implementation=]. A user agents
```
Comment (reillyeon) [inline index.html:751]:
```suggestion
          [=implementation-defined|left to the implementation=]. A user agent
```

File: index.html [hunk 2]
```diff
@@ -747,9 +747,9 @@ <h2>
...
+          MAY impose a rate limit on the frequency position changes.
```
Comment (reillyeon) [inline index.html:752]:
```suggestion
          MAY impose a rate limit on the frequency of position changes.
```

**Review by reillyeon — APPROVED** [review]
LGTM with some fixes to the existing language.

**Comment by marcoscaceres** [comment]
Thanks for the fixes... sorry for my dyslexia :(

**Pull request merged**